qa/tasks/cephfs/test_volumes.py

```diff
@@ -649,6 +649,20 @@ def test_volume_info(self):
         self.assertEqual(vol_info["used_size"], 0,
                          "Size should be zero when volumes directory is empty")
 
+    def test_volume_info_pending_subvol_deletions(self):
+        """
+        Tests the pending_subvolume_deletions in 'fs volume info' command
+        """
+        subvolname = self._generate_random_subvolume_name()
+        # create subvolume
+        self._fs_cmd("subvolume", "create", self.volname, subvolname)
```

Comment:
`subvolume create` needs `--mode` argument to be set, otherwise `_do_subvolume_io` won't execute

Reply:
Fixed it.